docs/examples/lambda-aws/index.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use MongoDB\Client;
+
+require_once __DIR__ . '/vendor/autoload.php';
+
+$uri = getenv('MONGODB_URI') ?: throw new RuntimeException('The MONGODB_URI environment variable is not set');
+$client = new Client($uri);
+$planets = $client
+    ->selectDatabase('sample_guides')
+    ->selectCollection('planets')
+    ->find(
+        [],
+        [
+            'sort' => ['orderFromSun' => 1],
+            'typeMap' => ['root' => 'array'],
+        ],
```

Comment:
With the default type map used in the library we can still refer to either `$planet['name']` or `$planet->name` below without specifying a different one. This makes the example a little more concise:
```suggestion
        ['sort' => ['orderFromSun' => 1]],
```